community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/CypherCompiler.scala

```diff
@@ -62,6 +62,8 @@ case class CypherCompilerConfiguration(queryCacheSize: Int,
                                        statsDivergenceThreshold: Double,
                                        queryPlanTTL: Long,
                                        useErrorsOverWarnings: Boolean,
+                                       idpMaxTableSize: Int,
+                                       idpIterationDuration: Long,
```

Comment:
maxDuration seemed to indicate the duration of the entire IDP process, not just an inner function inside the compaction loop. I choose 'iteration' over 'max' for this reason, and also though 'idpMaxDurationOfIteration' too ugly... :-)